diesel_migrations/migrations_macros/src/embed_migrations.rs

```diff
@@ -22,11 +22,8 @@ pub fn expand(path: String) -> proc_macro2::TokenStream {
     quote! {
         #[allow(dead_code)]
         mod embedded_migrations {
-            extern crate diesel;
```

Comment:
The point of those references are that a user does not necessarily have `diesel` imported as diesel. They can have something like `extern crate diesel as diesel_something` or have renamed diesel via a `Cargo.toml` option.

Reply:
> have renamed diesel via a Cargo.toml option.

Correct me if I am wrong, but in that case, wouldn't this line give a compiler error?

Reply:
Just to write that somewhere: I try to avoid random changes to just adopt some newer pattern there as long as they are only stylistic and don't improve the code in other ways as you loose the ability to just to a `git blame` to see why something was implemented that way.